Remove the build skuba command from testrunner

chentex opened the pull request:
It makes no sense to have this in here since we already have a
build or install target in the root's Makefile.

It is also giving testrunner more responsibilities that it should
have.

Also does some code format refactor.

bergmannf commented on ci/infra/testrunner/testrunner.py:
```diff
@@ -7,17 +7,19 @@
-from argparse import (ArgumentParser, REMAINDER)
+from argparse import REMAINDER, ArgumentParser
+
+from utils import BaseConfig, Logger, Utils
```
Alternatively we could decide to go with something like https://github.com/timothycrosley/isort, which will sort imports for us, so we no longer have to worry about import ordering.

chentex commented:
> I agree with the change, we should be calling skuba commands as much as possible. I would have liked to see the whitespace changes in a separate commit, though.
> 

Hi @stefsuse I have addressed your comments sorry about the mix of spaces and code refactor.

> Anything marked (future) is a suggestion not really about your change.

Including the future suggestions
